docs/adr/adr-010-incentivized-testnet-monitoring.md

```diff
@@ -0,0 +1,230 @@
+# ADR #010: Incentivized Testnet Monitoring
+
+## Changelog
+
+- 2022-7-19: Started
+- 2022-7-22: Add section on "How to monitor celestia-node with Grafana Cloud"
+- 2022-7-26: Add section on "How to monitor celestia-node with Uptrace"
+- 2022-7-29: Add section on "How to send data over HTTPS"
+
+## Context
+
+We're adding telemetry to celestia-node by instrumenting our codebase with metrics (see [ADR-009-telemetry](./adr-009-telemetry.md)). If the option to report metrics is enabled on celestia-node, then celestia-node will push metrics via [OLTP Exporter](https://opentelemetry.io/docs/reference/specification/protocol/exporter/) to an [OTEL Collector](https://opentelemetry.io/docs/collector/) instance. The Celestia team will manage an OTEL Collector instance during the incentivized testnet (and likely beyond) to collect metrics from nodes on the network. Celestia-node operators have the option of running their own OTEL Collector instance in order to collect metrics from their nodes.
+
+We would like to make the metrics collected in the Celestia team managed OTEL Collector actionable by making them queryable in internal Grafana dashboards. We additionally want a subset of metrics to be queryable by a public incentivized testnet leaderboard frontend.
+
+We would like to make it possible for node operators to monitor their own nodes with the telemetry tools Grafana and Uptrace. They can achieve this by deploying their own OTEL Collector instance and configuring it to export metrics to Grafana or Uptrace.
+
+This document proposes a strategy for making data in the Celestia team managed OTEL Collector available for use in internal Grafana dashboards and a public leaderboard. Additionally it describes how a node operator can monitor their own node by deploying their own OTEL Collector instance.
+
+![Incentivized Testnet Monitoring Diagram](./img/incentivized-testnet-monitoring-diagram.svg)
+
+// Comment on diagram <https://lucid.app/lucidchart/d957570f-9c06-4a82-9843-00d8232f734a/edit?invitationId=inv_247247d6-3a67-40b1-8513-1818caadd627>#
```

Comment:
Maybe change to: diagram created via https://lucid.app/lucidchart/d957570f-9c06-4a82-9843-00d8232f734a/edit?invitationId=inv_247247d6-3a67-40b1-8513-1818caadd627 
or sth like that.

Reply:
I expect the diagram won't be edited or commented on (in lucidchart) after this PR merges so I'm inclined to delete the link entirely. If reviewers have comments on the diagram while this PR is still in review, feel free to comment in this PR and I can edit the diagram and upload a revised diagram.